### DISCUSSION TITLE
Suggestions for Interests

### DISCUSSION THREAD

**Discussion by ghost** [Automattic/gravatar#76] [Ideas] [answered]
I want to see Numerology as an Interest we can add to our Gravatar profiles. Astrology is the closest thing for now and it just won't do... 😞 

This thread can also serve as a means for other uses to request Interests to add, so let the Gravatar team know what Interests you want added below.

**Comment by johngodley** [comment, marked as answer]
You can actually type in your own interests! Just type into the field but don't pick anything from the dropdown. The press return and it will be saved as a new interest

**Comment by blackEberechi** [comment]
> I want to see Numerology as an Interest we can add to our Gravatar profiles. Astrology is the closest thing for now and it just won't do... 😞
> 
> This thread can also serve as a means for other uses to request Interests to add, so let the Gravatar team know what Interests you want added below.

this is a nice idea

**Comment by aaronfc** [comment]
Closing discussion as this can already be achieved [as described by John above](https://github.com/Automattic/gravatar/discussions/76#discussioncomment-10624356).
